### PR TITLE
Fix burn in bindings

### PR DESCRIPTION
**Pull request by DaughterOfMars**
# Description of change

These structures were out of date somehow.

### CODE REVIEW

File: bindings/python/iota_sdk/types/burn.py [hunk 1]
```diff
@@ -25,8 +25,9 @@ class Burn:
     mana: Optional[bool] = None
     generated_mana: Optional[bool] = None
     accounts: Optional[List[HexStr]] = None
-    nfts: Optional[List[HexStr]] = None
     foundries: Optional[List[HexStr]] = None
+    nfts: Optional[List[HexStr]] = None
+    delegations: Optional[List[HexStr]] = None
```
Comment (Thoralf-M) [inline bindings/python/iota_sdk/types/burn.py:31]: Docs not updated for delegations

Comment (DaughterOfMars) [reply bindings/python/iota_sdk/types/burn.py:31]: honestly, how did I miss that 😭

File: bindings/python/iota_sdk/types/burn.py [hunk 2]
```diff
@@ -19,14 +19,16 @@ class Burn:
         accounts: The accounts to burn.
         nfts: The NFTs to burn.
```
Comment (thibault-martinez) [inline bindings/python/iota_sdk/types/burn.py]: Can you also fix the order here pls?

Comment (DaughterOfMars) [reply bindings/python/iota_sdk/types/burn.py]: done

**Review by thibault-martinez — APPROVED** [review]

**Review by Thoralf-M — APPROVED** [review]

**Pull request merged**